packages/compass-connections/src/provider.ts

```diff
@@ -1 +1,61 @@
+import { createContext, useContext } from 'react';
+import { createServiceLocator } from 'hadron-app-registry';
+import { useConnectionInfo } from '@mongodb-js/connection-storage/provider';
+
+import type { DataService } from 'mongodb-data-service';
```

Comment:
Let's re-export the type too so that you don't need to add two dependencies when using data service locator